packages/traceloop-sdk/tests/dataset/mock_objects.py

```
@@ -0,0 +1,130 @@
+            "cmdr3ce1s0004hmp0ies575jr": 52,
+            "cmdr3ce1s0005hmp0bdln01js": False,
+        },
+    ]
+
+
+def setup_mock_http_client_for_get_rows():
+    """Setup mock HTTP client for getting existing rows"""
+    mock_client = MagicMock()
+    mock_client.post.side_effect = [
+        create_dataset_response(),  # create_dataset call
+        json.loads(create_rows_response_json),  # get existing rows
```

Ruff (0.12.2)

Undefined name `json`

(F821)